script.js

```diff
@@ -130,12 +130,17 @@ closeElement.addEventListener('click', () => {
   popup.style.display = 'none';
 });
 
-// const email = document.getElementById('mail');
-
-// email.addEventListener('input', (event) => {
-// if (email.validity.typeMismatch) {
-///  email.setCustomValidity('please use an E-mail address!');
-// } else {
-//   email.setCustomValidity('');
-// }
-// });
+const form = document.getElementById('input-form');
+// console.log('form: ', form);
```

Comment:
### Optional
- [ ] It would be better if we remove all commented piece of code, this makes our codebase look messy